refactor/imports.go

```diff
@@ -38,11 +38,12 @@ func deleteUnusedImports(s *Snapshot, p *Package, text []byte) []byte {
 
 	match := func(name, pkg string) bool {
 		if name == "" {
-			p1 := s.pkgGraph.byPath(pkg)
```

Comment:
Maybe something like this? I like how in snap.go we first determine the correct path to look up in the pkggraph and then look it up. 

```
path := pkg
if path1, ok := p.ImportMap[path]; ok {
	path = path1
}

p1 := s.pkgGraph.byPath(path)
```